modules/nw-metallb-operator-setting-runtimeclass.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-operator-install.adoc
+
+[id="nw-metallb-operator-setting-runtimeclass_{context}"]
+= Configuring a container runtime class in a MetalLB deployment
+
+You can optionally assign a container runtime class to `controller` and `speaker` pods by configuring the MetalLB custom resource. For example, for Windows workloads, you can assign a Windows runtime class to the pod, which uses this runtime class for all containers in the pod.  
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Install the MetalLB Operator.
+
+.Procedure
+. Create a `RuntimeClass` by using a custom resource:
++
+[source,yaml,options="nowrap",role="white-space-pre"]
+----
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: myclass 
+handler: myconfiguration
+----
+
+. Specify the `RuntimeClass` value in the MetalLB custom resource: 
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: MetalLB
+metadata:
+  name: metallb
+  namespace: metallb-system
+spec:
+  logLevel: debug
+  controllerConfig:
+    runtimeClassName: myclass
```

Comment:
Are there any other guidelines or limitations to the `runtimeClassName` values?

Reply:
`runtimeClassName` accepts a string which is the name of the runtime class you defined in the previous step